### PR TITLE
name changes

### PR DESCRIPTION
**Pull request by djrtwo**
- [x] penalized -> slashed
- [x] exit -> voluntary exit
- [x] operations -> transactions
- [x] initial, startup -> genesis
- [x] `ChainStart` -> `Eth2Genesis`
- [x] Remove `Number` from custom types 
- [x] `SEED_LOOKAHEAD` -> `MIN_SEED_LOOKAHED`
- [x] `ENTRY_EXIT_DELAY` -> `ACTIVATION_EXIT_DELAY`
- [x] `INCLUDER_REWARD_QUOTIENT` -> `ATTESTATION_INCLUSION_REWARD_QUOTIENT`
- [x] `latest_index_roots` -> `latest_active_index_roots`
- [x] `current_epoch_start_shard` -> `current_shuffling_start_shard`, `current_shuffling_epoch`, `current_shuffling_seed`
- [x] `MIN_VALIDATOR_WITHDRAWAL_EPOCHS` -> `MIN_VALIDATOR_WITHDRAWAL_DELAY`
- [x] `MAX_WITHDRAWALS_PER_EPOCH` -> `MAX_EXIT_DEQUEUES_PER_EPOCH`
- [x] `ETH1_DATA_VOTING_PERIOD` -> `EPOCHS_PER_ETH1_VOTING_PERIOD`
- [x] `SLOT_DURATION` -> `SECONDS_PER_SLOT`
- [x] `EPOCH_LENGTH` -> `SLOTS_PER_EPOCH`

**Exits/penalties clean-up recap**

* exits
	* voluntary exits
	* forceful exits
		* slashings
		* ejections
* penalties
	* base penalties
	* inactivity penalties
	* (partial) slashings

### CODE REVIEW

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -606,14 +606,14 @@ We define the following Python custom types for type hinting and readability:
 
 | Name | SSZ equivalent | Description |
 | - | - | - |
-| `SlotNumber` | `uint64` | a slot number |
-| `EpochNumber` | `uint64` | an epoch number |
-| `ShardNumber` | `uint64` | a shard number |
+| `Slot` | `uint64` | a slot number |
+| `Epoch` | `uint64` | an epoch number |
+| `Shard` | `uint64` | a shard number |
```
Comment (hwwhww) [inline specs/core/0_beacon-chain.md]: The reason why I named it `ShardNumber` is that I assumed that we will have other ID schema for identifying the shard chain, maybe `ShardID`.

Comment (djrtwo) [reply specs/core/0_beacon-chain.md]: I see... I think in terms of the spec, we would treat those as the same type and value. I understand implementation might choose to use two types.

**Comment by djrtwo** [comment]
cc: @vbuterin regarding "operations -> transactions".

**Comment by JustinDrake** [comment]
😂 

screenshot 2019-02-12 at 19 48 39

**Comment by JustinDrake** [comment]
Substrates calls the operations/transactions "extrinsics". See https://www.youtube.com/watch?v=GDvV-Vn2u3w&feature=youtu.be&t=928

**Comment by prestonvanloon** [comment]
FYI: we are not renaming operations to transactions in Prysm. We generally think this should be reserved for user level and operations is the more intuitive name. 

We are also not renaming chainstart to eth2genesis. For the same reasons: more intuitive with the previous name.

**Pull request merged**